### PR TITLE
Do not allow a user to increment their own karma

### PR DESCRIPTION
**Pull request by mushfick**
This will stop users (mainly @tielur) from incrementing their own karma.

### CODE REVIEW

File: lib/alice/handlers/karma.ex [hunk 1]
```diff
@@ -46,11 +53,15 @@ defmodule Alice.Handlers.Karma do
     |> reply("#{term} has had its karma scattered to the winds. (#{count})")
   end
 
+  defp _increment(term, term, conn), do: reply(conn, ~s(#{@karma_denial_message}))
```
Comment (mattr-) [inline lib/alice/handlers/karma.ex]: You don't need the underscore for private methods. Just the fact that it's a different arity is good enough.

**Review by mattr- — CHANGES_REQUESTED** [review]
Just the one change to remove the underscore and then I think we're good here.

File: lib/alice/handlers/karma.ex (hunk 1 above)
Comment (mushfick) [reply lib/alice/handlers/karma.ex]: Makes sense. I mainly did it to be explicit about the helper function.

**Review by mattr- — CHANGES_REQUESTED** [review]
:boom: done

**Review by mattr- — APPROVED** [review]

File: lib/alice/handlers/karma.ex [hunk 2]
```diff
@@ -46,11 +53,15 @@ defmodule Alice.Handlers.Karma do
     |> reply("#{term} has had its karma scattered to the winds. (#{count})")
   end
 
+  defp increment(term, term, conn), do: reply(conn, ~s(#{@karma_denial_message}))
```
Comment (tielur) [inline lib/alice/handlers/karma.ex:56]: Isn't the 2nd parameter a `user`?

Comment (mushfick) [reply lib/alice/handlers/karma.ex:56]: It is but we're using pattern matching here to say if the `user` that is passed along is equal to `term` then call `reply..`

Comment (adamzaninovich) [reply lib/alice/handlers/karma.ex:56]: they should both be `user` then

Comment (mushfick) [reply lib/alice/handlers/karma.ex:56]: sorry i meant, we're using pattern matching here to say if the `term` that is getting it's karma incremented is the same as the `user`, then reply with the denial message

**Pull request merged**